apps/src/lib/node/ledger/events/log/dumb_queries.rs

```diff
@@ -0,0 +1,155 @@
+//! Silly simple Tendermint query parser.
```

Comment:
Will we still need `dumb_queries.rs` or use the approach of having `Path` `Accepted`/`Applied` enum variants you mentioned? (in https://github.com/anoma/namada/blob/a669ca573f1d3f61b87cdcb8eba7cb9a6f872a6a/apps/src/lib/node/ledger/rpc.rs)

Reply:
they're still needed, but I added two new methods to directly instantiate a matcher cd6746a4f31fa620495c7b1c0c846df12ee3a1ac